src/js/plugins/utils.ts

```diff
@@ -6,7 +6,8 @@ export const getPluginName = function(url: string): string {
     return url.replace(/^(.*\/)?([^-]*)-?.*\.(js)$/, '$2');
 };
 
-export function getPluginErrorCode(/* pluginURL: string */): number {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function getPluginErrorCode(pluginURL: string): number {
```

Comment:
Do we need to add this back if the commercial method is being used instead? Or is it more so for consistency?

Reply:
Since we're passing it in in the OSS loader, typescript's typecheck fails because it's passing an arg to a function that doesn't take any in :(

Reply:
ahhh. oh typescript.